include/CL/sycl/vec.hpp

```diff
@@ -80,14 +80,23 @@ namespace cl::sycl {
   /** Accessors to access hex indexed elements of a vector
    */
 #define TRISYCL_DECLARE_S(x)                                           \
-  DataType& s##x() {                                                    \
-    return (*this)[(0x##x)];                                            \
-  }
-
-#define TRISYCL_GEN_SWIZ2(str,idx0,idx1) __swizzled_vec__<DataType, 2> str() const { return base_vec::swizzle(idx0, idx1); }
-#define TRISYCL_GEN_SWIZ3(str,idx0,idx1,idx2) __swizzled_vec__<DataType, 3> str() const { return base_vec::swizzle(idx0, idx1, idx2); }
-#define TRISYCL_GEN_SWIZ4(str,idx0,idx1,idx2,idx3) __swizzled_vec__<DataType, 4> str() const { return base_vec::swizzle(idx0, idx1, idx2, idx3); }
-
+  const DataType& s##x() const {                                       \
+    return (*this)[(0x##x)];                                           \
+  }                                                                    \
+                                                                       \
+  DataType& s##x() {                                                   \
+    return (*this)[(0x##x)];                                           \
+  }
+
+#define TRISYCL_GEN_SWIZ2(str,idx0,idx1)                                                    \
+  const __swizzled_vec__<DataType, 2> str() const { return base_vec::swizzle(idx0, idx1); } \
```

Comment:
Lines too long.
Add newline after `{` and before  `}`
Same for all lines up to line 99.